python/src/libint2/libint2.cc

```diff
@@ -137,6 +137,12 @@ std::vector<double> coeffs_normalized(const Shell &s) {
 
 PYBIND11_MODULE(libint2, m) {
 
+  py::enum_<SHGShellOrdering>(m, "SHGShellOrdering")
+    .value("SHGShellOrdering_Standard", libint2::SHGShellOrdering_Standard)
+    .value("SHGShellOrdering_Gaussian", libint2::SHGShellOrdering_Gaussian)
+    .value("SHGShellOrdering_MOLDEN", libint2::SHGShellOrdering_Gaussian)
```

Comment:
On further investigation, I don't think this is a typo. The Molden: Gaussian mirrors https://github.com/evaleev/libint/blob/master/include/libint2/shgshell_ordering.h#L33 , and the defines in that file do seem to match, too. 

I'm guessing it was a convenience to have Molden always defined even when the main SH value toggled.